pages/news/_slug.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <article class="pa-4">
+    <h1 class="text-uppercase">{{ article.title }}</h1>
+    <p class="grey--text text--darken-1">
+      Updated: {{ formatDate(article.updatedAt) }}
+    </p>
+    <p class="font-italic">{{ article.description }}</p>
```

Comment:
nit: could just use `font-style: italic;`

Reply:
You mean down in the style section, right?